Product/Production/Adapters/General/CONNECTDirectConfig/src/main/java/gov/hhs/fha/nhinc/directconfig/processor/impl/DefaultBundleRefreshProcessorImpl.java

```diff
@@ -394,10 +401,10 @@ public void refreshBundle(TrustBundle bundle) {
             retVal = outStream.toByteArray();
         } catch (SocketTimeoutException e) {
             dao.updateLastUpdateError(bundle.getId(), processAttempStart, BundleRefreshError.DOWNLOAD_TIMEOUT);
-            log.warn("Failed to download bundle", e);
+            LOG.warn("Failed to download bundle", e);
```

Comment:
As part of this PR, I don't plan to change log levels that were already in place.  This belongs as part of a Direct Config refactor effort.